maxim/tests/test_prompts.py

```
@@ -41,15 +41,15 @@ def tearDown(self):
     def test_get_prompt_with_deployment_variables(self):
```

Ruff (0.12.2)

Missing return type annotation for public function `test_get_prompt_with_deployment_variables`

Add return type annotation: `None`

(ANN201)